### PR TITLE
Remove ethereum from ChainSelector

### PR DESCRIPTION
**Pull request by leanazulyoro**
This removes Ethereum from the ChainSelector and leaves only **Hemi Sepolia**.

Captura de pantalla 2024-04-15 a la(s) 09 49 33

It also removes the "buy" tab. It just hides the button, the whole feature's components are still available if needed in the future.
Regarding the "limit" tab, it appears to be under a feature flag, and disabled, I left it like that.

### CODE REVIEW

File: apps/web/src/connection/index.ts [hunk 1]
```diff
@@ -21,6 +21,8 @@ import { Connection, ConnectionType, ProviderInfo } from './types'
 import { getDeprecatedInjection, getIsCoinbaseWallet, getIsInjected, getIsMetaMaskWallet } from './utils'
 import { UniwalletConnect as UniwalletWCV2Connect, WalletConnectV2 } from './WalletConnectV2'
 
+const DEFAULT_CHAIN_ID = ChainId.HEMI_SEPOLIA
```
Comment (gabmontes) [inline apps/web/src/connection/index.ts:24]: Should we place this in a "global config" so it is easier to change once i.e. we switch to Hemi mainnet?

File: apps/web/src/components/Logo/ChainLogo.tsx [hunk 2]
```diff
@@ -1,33 +1,18 @@
-import { ChainId } from '@uniswap/sdk-core'
+import { ChainId } from '@hemilabs/sdk-core'
```
Comment (gndelia) [inline apps/web/src/components/Logo/ChainLogo.tsx]: This shouldn't be necessary, as we're fixing this on build time on [tsconfig](https://github.com/hemilabs/interface/blob/b48f3a38b18ceb7e6876fcdac085a00be39420ad/apps/web/tsconfig.json#L32) and [craco.config.cjs](https://github.com/hemilabs/interface/blob/b48f3a38b18ceb7e6876fcdac085a00be39420ad/apps/web/craco.config.cjs#L94). 
Do you get any errors?

Comment (leanazulyoro) [reply apps/web/src/components/Logo/ChainLogo.tsx]: Right, it's not needed. Changed here: https://github.com/hemilabs/interface/pull/3/commits/86e5c778ba600c823948676982198f575c69d51b

File: apps/web/src/connection/index.ts (hunk 1 above)
Comment (leanazulyoro) [reply apps/web/src/connection/index.ts:24]: I replaced one extra `ChainId.HEMI_SEPOLIA` that was in this same file below with this DEFAULT_CHAIN_ID so we only have this one line to change (here: https://github.com/hemilabs/interface/pull/3/commits/86e5c778ba600c823948676982198f575c69d51b).
This sets the list of chains on the ChainSelector only, I'd leave this here for now. I suggest we wait till we have HEMI mainnet and see then to which extend we can have one centralized config. Right now, this is too widely spread on many places, including external packages.

**Review by gndelia — APPROVED** [review]

**Review by gabmontes — APPROVED** [review]

**Pull request merged**